src/bsoncxx/stdx/make_unique.hpp

```diff
@@ -48,7 +48,7 @@ using ::boost::make_unique;
 BSONCXX_INLINE_NAMESPACE_END
 }  // namespace bsoncxx
 
-#elif __cplusplus >= 201402L
+#elif __cplusplus >= 201402L || (defined(_MSVC_LANG) && _MSVC_LANG >= 201402L)
```

Comment:
Was concerned about testing `__cplusplus` before `_MSVC_LANG`, but it seems [Catch2](https://github.com/catchorg/Catch2/blob/e932bcf7a349f1c2fd02369df8c923afccb4638c/src/catch2/internal/catch_compiler_capabilities.hpp#L30) does this as well, so I believe this should be fine.